platform-dom/src/main/java/com/softicar/platform/dom/elements/number/decimal/DomDecimalDisplay.java

```diff
@@ -0,0 +1,54 @@
+package com.softicar.platform.dom.elements.number.decimal;
+
+import com.softicar.platform.common.core.locale.CurrentLocale;
+import com.softicar.platform.common.core.number.BigDecimalMapper;
+import com.softicar.platform.common.core.number.formatter.BigDecimalFormatter;
+import com.softicar.platform.common.core.number.formatter.BigDecimalScaleApplier;
+import com.softicar.platform.dom.element.IDomElement;
+import com.softicar.platform.dom.elements.DomDiv;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+/**
+ * An {@link IDomElement} to display decimal values according to
+ * {@link CurrentLocale}.
+ *
+ * @author Oliver Richers
+ */
+public class DomDecimalDisplay<T extends Number> extends DomDiv {
```

Comment:
I think this element deserves its own unit test, considering its importance: We'll use it to display values all over the place, in ERP software. So we should ensure correct rounding/scaling/truncation/whatever behavior.

Reply:
added DomDecimalDisplayTest **and** BigDecimalScaleApplierTest